champions/src/main/java/me/mykindos/betterpvp/champions/champions/builds/RoleBuild.java

```diff
@@ -145,4 +145,35 @@ public Component getBuildComponent() {
         return component;
     }
 
+    /**
+     * Returns true if the role and BuildSkills are all equal
+     * @param o the object to check against
+     * @return true if this object has the same build
+     */
+    public boolean buildEquals(Object o) {
```

Comment:
Shouldn't this just be overriding equals? doesnt seem to be getting used otherwise

Reply:
I think what I was going to use this for I didn't end up implementing. This method explicity compares builds without regard to the player or build #, which an equals should do. But can be removed.